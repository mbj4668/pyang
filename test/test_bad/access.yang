module access {
    namespace urn:test:access;
    prefix a;
    description "Should be made unreadable prior to testing";
}
