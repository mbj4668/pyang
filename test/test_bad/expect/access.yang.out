error access.yang: [Errno 13] Permission denied: 'access.yang'
